README.md

```diff
@@ -135,21 +135,46 @@ logging_inputs:
     type: basics
 ```
 
-3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and remote rsyslog and outputs into the local files.
+3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the remote rsyslog and outputs into the per host base local files.
 ```
-logging_enabled: true
-rsyslog_capabilities: [ 'network', 'remote-files' ]
-logging_purge_confs: true
-logging_outputs:
-  - name: local-files
-    type: files
-logging_inputs:
-  - name: system-and-remote-input
-    type: basics
-logging_flows:
-  - name: flow0
-    inputs: [system-and-remote-input]
-    outputs: [local-files]
```

Comment:
is there a reason to increase the indent of those lines?

Reply:
No, I don't think so...